src/entt/signal/dispatcher.hpp

```diff
@@ -34,6 +34,7 @@ class dispatcher {
         virtual void publish() = 0;
         virtual void disconnect(void *) = 0;
         virtual void clear() ENTT_NOEXCEPT = 0;
+        virtual size_t size() = 0;
```

Comment:
```suggestion
        virtual std::size_t size() const ENTT_NOEXCEPT = 0;
```
I tend to use `std::size_t` rather than `size_t`. Moreover, this should be `const ENTT_NOEXCEPT` probably.